ansible_collections/arista/avd/molecule/eos_designs_deprecated_vars/inventory/host_vars/host1/port-channel-short-esi.yml

```diff
@@ -26,6 +18,8 @@ servers:
         mtu: 1601
```

Comment:
I cannot insert the comment in the right line, but the `switches` key in line 17 should also be updated with `host1` to lead to any config.

Reply:
fixed